tensorflow/core/kernels/data/concatenate_dataset_op_test.cc

```diff
@@ -384,6 +384,7 @@ TEST_F(ConcatenateDatasetOpTest, DatasetSave) {
   DatasetBase *concatenate_dataset;
   TF_ASSERT_OK(CreateDataset(dataset_kernel.get(), dataset_kernel_ctx.get(),
                              &concatenate_dataset));
+  core::ScopedUnref scored_unref(concatenate_dataset);
```

Comment:
Any reason this is called "scored" as opposed to "scoped"? If not, please rename this here and anywhere else this typo occurs.

Reply:
@jsimsa This is a typo issue. The typos occurred here and other places are fixed https://github.com/tensorflow/tensorflow/pull/26860/commits/058d1abaf3f110b7e3c5ebadae141bd6f4eb1ef2.